elliottlib/imagecfg.py

```diff
@@ -0,0 +1,52 @@
+import assertion
+import exectools
+import logutil
+from metadata import Metadata
+
+logger = logutil.getLogger(__name__)
+
+
+class ImageMetadata(Metadata):
+
+    def __init__(self, runtime, data_obj):
+        super(ImageMetadata, self).__init__('image', runtime, data_obj)
+        self.image_name = self.config.name
+        self.image_name_short = self.image_name.split('/')[-1]
+
+    @property
+    def base_only(self):
+        """
+        Some images are marked base-only.  Return the flag from the config file
+        if present.
+        """
+        return self.config.base_only
+
+    def get_latest_build_info(self):
```

Comment:
This has recently been updated and now takes the `product_version` param. Start with updated `image.py` -- this breaks image finds:
```
🐚 elliott -g openshift-4.1 find-builds -k image
2019-10-31 04:34:18,179 INFO Cloning config data from https://github.com/openshift/ocp-build-data.git
2019-10-31 04:34:18,678 INFO Using branch from group.yml: rhaos-4.1-rhel-7
Generating list of images: Hold on a moment, fetching Brew buildinfo
[********************************************************************************************************************]
[******Traceback (most recent call last):
...
  File "/home/lmeyer/.local/lib/python2.7/site-packages/elliottlib/cli/find_builds_cli.py", line 112, in find_builds_cli
    unshipped_builds = _fetch_builds_by_kind_image(runtime, product_version, session)
  File "/home/lmeyer/.local/lib/python2.7/site-packages/elliottlib/cli/find_builds_cli.py", line 187, in _fetch_builds_by_kind_image
    lambda build: build.get_latest_build_info(product_version_overide)
  File "/home/lmeyer/.local/lib/python2.7/site-packages/elliottlib/util.py", line 229, in parallel_results_with_progress
    inputs)
  File "/usr/lib64/python2.7/multiprocessing/pool.py", line 253, in map
    return self.map_async(func, iterable, chunksize).get()
  File "/usr/lib64/python2.7/multiprocessing/pool.py", line 572, in get
    raise self._value
TypeError: get_latest_build_info() takes exactly 1 argument (2 given)
```

Reply:
Fixed